README.md

```diff
@@ -272,6 +273,15 @@ The migration process currently consists of 3 commands:
 
 ---
 
+## Debugging with Pycharm
```

Comment:
It took me a couple hours to get all the configs and order of operations right for Pycharm to work for me. I'm documenting it for the sake of future devs/future me.

Reply:
I agree with @Niicck on this one, I went down a more complex path than this set of instructions when trying to get everything working with the pycharm debugger and I'm not sure what exactly was "magic" because I just happened to have my env set up in ways that worked and what was actual intentional.

If we want to change it to say, "Debugging with an IDE" and then have a subheading of "Pycharm" and a "TODO" for other IDEs that could work, but I'm very glad to see these instructions make it into the readme.